libtextsecure/sendmessage.js

```diff
@@ -273,6 +286,8 @@ window.textsecure.messaging = function() {
                 else
                     resolve();
             });
+        }).then(function() {
+            return sendSyncMessage(proto, timestamp, getString(proto.group.id));
```

Comment:
do we really mangle the destination field as /either/ a valid groupid or a valid individual identifier? RIght now we can identify by things that start with "+", but I dont think we want to rely on that in the future? @moxie0 might want to comment here.